client/src/locales/en-US.json

```diff
@@ -995,6 +995,17 @@
                 "description": "Can only access shared workspaces."
             }
         },
+        "updateProfile": {
+            "success": "The profile has been updated! | The profiles have been updated!",
+            "failedOffline": "Could not update the profile. Make sure you are online. | Could not update the profiles. Make sure you are online.",
+            "failedGeneric": "Could not update the profile. | Could not update the profiles.",
+            "actionUpdate": "Update",
+            "actionCancel": "Cancel",
+            "title": "Update the profile | Update the profiles",
```

Comment:
I'd change `Update` to `Change`, since it's a modification and not an update, and change the button text from `Update` to `Confirm`